@NitzanMordhai - Getting following error while creating shaman build-
```
Merging PR #69230
Fetching 6c6fc7289257db2929378b58384bc244d62e2c36 from https://github.com/ceph/ceph.git
Now have head for PR #69230: 6c6fc7289257db2929378b58384bc244d62e2c36
[ 6c6fc728 ] Ceph tracker cited: Fixes: https://tracker.ceph.com/issues/76962
Traceback (most recent call last):
  File "/Users/skanta/Office/upstream/ptl-tool/ceph-ptl/ptl-tool.py", line 2518, in <module>
    main()
    ~~~~^^
  File "/Users/skanta/Office/upstream/ptl-tool/ceph-ptl/ptl-tool.py", line 2515, in main
    return build_branch(args)
  File "/Users/skanta/Office/upstream/ptl-tool/ceph-ptl/ptl-tool.py", line 2191, in build_branch
    G.git.merge(tip.hexsha, '--no-ff', m=message)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/skanta/Office/upstream/ptl-tool/ptl-venv/lib/python3.13/site-packages/git/cmd.py", line 1087, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
                                   ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/skanta/Office/upstream/ptl-tool/ptl-venv/lib/python3.13/site-packages/git/cmd.py", line 1744, in _call_process
    return self.execute(call, **exec_kwargs)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/Users/skanta/Office/upstream/ptl-tool/ptl-venv/lib/python3.13/site-packages/git/cmd.py", line 1534, in execute
    raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git merge -m Merge PR #69230 into wip-bharath5-testing-20260804.031944-main

* refs/pull/69230/head:
	suites/upgrade: add but still running to ignorelist

 6c6fc7289257db2929378b58384bc244d62e2c36 --no-ff
  stdout: 'Auto-merging qa/suites/upgrade/squid-x/parallel/0-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/squid-x/parallel/0-start.yaml
Auto-merging qa/suites/upgrade/squid-x/stress-split/1-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/squid-x/stress-split/1-start.yaml
Auto-merging qa/suites/upgrade/telemetry-upgrade/squid-x/1-tasks.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/telemetry-upgrade/squid-x/1-tasks.yaml
Auto-merging qa/suites/upgrade/telemetry-upgrade/tentacle-x/1-tasks.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/telemetry-upgrade/tentacle-x/1-tasks.yaml
Auto-merging qa/suites/upgrade/tentacle-x/parallel/0-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/tentacle-x/parallel/0-start.yaml
Auto-merging qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
Automatic merge failed; fix conflicts and then commit the result
```

Removing the needs-qa label for now. Please add the label back once the fix has been implemented .

--- a/qa/suites/upgrade/squid-x/parallel/0-start.yaml
+++ b/qa/suites/upgrade/squid-x/parallel/0-start.yaml
@@ -61,3 +61,4 @@ overrides:
       - POOL_FULL
       - pool full
       - pool\(s\) full
+      - but it is still running

--- a/qa/suites/upgrade/squid-x/stress-split/1-start.yaml
+++ b/qa/suites/upgrade/squid-x/stress-split/1-start.yaml
@@ -39,6 +39,7 @@ overrides:
       - POOL_FULL
       - pool full
       - pool\(s\) full
+      - but it is still running
 tasks:
 - install:
     branch: squid

--- a/qa/suites/upgrade/telemetry-upgrade/squid-x/1-tasks.yaml
+++ b/qa/suites/upgrade/telemetry-upgrade/squid-x/1-tasks.yaml
@@ -30,6 +30,7 @@ overrides:
       - pool full
       - pool\(s\) full
       - telemetry module includes new collections
+      - but it is still running
 tasks:
 - install:
     branch: squid

--- a/qa/suites/upgrade/telemetry-upgrade/tentacle-x/1-tasks.yaml
+++ b/qa/suites/upgrade/telemetry-upgrade/tentacle-x/1-tasks.yaml
@@ -30,6 +30,7 @@ overrides:
       - pool full
       - pool\(s\) full
       - telemetry module includes new collections
+      - but it is still running
 tasks:
 - install:
     branch: tentacle

--- a/qa/suites/upgrade/tentacle-x/parallel/0-start.yaml
+++ b/qa/suites/upgrade/tentacle-x/parallel/0-start.yaml
@@ -62,3 +62,4 @@ overrides:
       - POOL_FULL
       - pool full
       - pool\(s\) full
+      - but it is still running

--- a/qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
+++ b/qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
@@ -39,6 +39,7 @@ overrides:
       - POOL_FULL
       - pool full
       - pool\(s\) full
+      - but it is still running
 tasks:
 - install:
     branch: tentacle

